components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java

```diff
@@ -817,20 +812,21 @@ public String getOpenIDRealm() {
     /**
      * @return
      */
-    public String getWstrustSP() {
-        if (wstrustEp != null) {
+    public List<String> getWstrustSP() {
```

Comment:
Shall we introduce a new public method and deprecate the old method? We should not change public methods as there can be usages in other places.

Reply:
Check the other changes as well.

Reply:
Solved in my last commit. I added a new method for this and kept the original one. Please review.